war/gulpfile.js

```diff
@@ -20,9 +27,9 @@ builder.tests('src/test/js');
 // Bundle the modules.
 // See https://github.com/tfennelly/jenkins-js-builder#bundling
 //
-builder.bundle('src/main/js/wizard.js')
+builder.bundle('src/main/js/pluginSetupWizard.js')
     .withExternalModuleMapping('jquery-detached', 'jquery-detached:jquery2')
-    .withExternalModuleMapping('jqueryui-detached', 'jquery-detached:jqueryui1')
+    .withExternalModuleMapping('bootstrap', 'bootstrap:bootstrap3')
     .withExternalModuleMapping('handlebars', 'handlebars:handlebars3')
```

Comment:
I'm afraid we've going to have to remove these (can do it later) and ram everything into an uber bundle.